src/utils.ts

```diff
@@ -73,46 +74,8 @@ export function getTokenFrequency(token: string, tokens: string[]): number {
 // MIT Licensed (https://github.com/lovasoa/fast_array_intersect/blob/master/LICENSE)
 // while on tag https://github.com/lovasoa/fast_array_intersect/tree/v1.1.0
 export function intersectTokenScores(arrays: TokenScore[][]): TokenScore[] {
-  if (arrays.length === 0) return [];
```

Comment:
Here I'd keep the original TS implementation as a fallback whenever WASM is not available/supported.
The idea is to create a simple function to detect the runtime, load WASM, and fallback to TS if not supported. WDYT?

Reply:
I totally agree.